automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/PFBImportSpec.scala

```diff
@@ -0,0 +1,186 @@
+
+        withCleanBillingProject(owner) { projectName =>
+          withWorkspace(projectName, prependUUID("writer-pfb-import"), aclEntries = List(AclEntry(writer.email, WorkspaceAccessLevel.Writer))) { workspaceName =>
+
+            // call importPFB as writer
+            val postResponse: String = Orchestration.postRequest(s"${workspaceUrl(projectName, workspaceName)}/importPFB", testPayload)(writerToken)
+            // expect to get exactly one jobId back
+            val importJobIdValues: Seq[JsValue] = postResponse.parseJson.asJsObject.getFields("jobId")
+            importJobIdValues should have size 1
+            val importJobId: String = importJobIdValues.head match {
+              case js:JsString => js.value
+              case x => fail("got in invalid jobId: " + x.toString())
+            }
+
+            // poll for completion as writer
+            eventually {
+              val resp = Orchestration.getRequest( s"${workspaceUrl(projectName, workspaceName)}/importPFB/$importJobId")(writerToken)
+              blockForStringBody(resp).parseJson.asJsObject.fields.get("status").value shouldBe JsString("Done")
+            }
+
+            // inspect data entities and confirm correct import as writer
+            eventually {
+              val resp = Orchestration.getRequest( s"${workspaceUrl(projectName, workspaceName)}/entities")(writerToken)
+              blockForStringBody(resp).parseJson shouldBe expectedEntities
+            }
+
+          } (ownerAuthToken)
+        }
+      }
+    }
+
+    "should asynchronously result in an error when attempting to import a PFB file via import service" - {
+      "if the file to be imported is invalid" in {
+        implicit val token: AuthToken = ownerAuthToken
+
+        withCleanBillingProject(owner) { projectName =>
+          withWorkspace(projectName, prependUUID("owner-pfb-import")) { workspaceName =>
+
+            // call importPFB as owner
+            val postResponse: String = Orchestration.postRequest(s"${workspaceUrl(projectName, workspaceName)}/importPFB",
+              Map("url" -> "https://storage.googleapis.com/fixtures-for-tests/fixtures/this-intentionally-does-not-exist"))
+            // expect to get exactly one jobId back
+            val importJobIdValues: Seq[JsValue] = postResponse.parseJson.asJsObject.getFields("jobId")
+            importJobIdValues should have size 1
+
+            val importJobId: String = importJobIdValues.head match {
+              case js:JsString => js.value
+              case x => fail("got in invalid jobId: " + x.toString())
+            }
+
+            // poll for completion as owner
+            eventually {
+              val resp: HttpResponse = Orchestration.getRequest( s"${workspaceUrl(projectName, workspaceName)}/importPFB/$importJobId")
+              resp.status shouldBe StatusCodes.OK
+              blockForStringBody(resp).parseJson.asJsObject.fields.get("status").value shouldBe JsString("Error")
```

Comment:
Nit: Is there a reason the error message here isn't more descriptive, unlike the other negative tests?

Reply:
The negative tests are looking at the text message contained in an `ErrorReport` object, which as you notice can be descriptive and human-readable. This test is looking only at the `status` value for the import job, which is an enum and constrained to specific values like `Done`, `Error`, and `Upserting`. I thought this test should only check that attempting to import a nonexistent file results in an error ... if Import Service changes its wording on the error message description, it won't break the test.